core/vm/logger.go

```diff
@@ -159,6 +161,126 @@ func (l *StructLogger) StructLogs() []StructLog {
 	return l.logs
 }
 
+// JavascriptTracer provides an implementation of Tracer that evaluates a
+// Javascript function for each VM execution step.
+type JavascriptTracer struct {
```

Comment:
Please move the tracer somewhere else.  The core/vm -> otto dependency is not nice. The best location is package internal/ethapi. You can create a new Go source file for it there.